spec.html

```diff
@@ -6956,6 +6996,28 @@ <h1>
       </emu-alg>
     </emu-clause>
 
+    <emu-clause id="sec-getiteratorflattenable" type="abstract operation">
+      <h1>
+        GetIteratorFlattenable (
```

Comment:
I would love to come up with a better name for this AO.

Reply:
This comment was about `GetIteratorFlattenable` (it's no longer displaying under the appropriate line in the diff view).